src/lua/LuaPlayer.cpp

```diff
@@ -721,7 +722,8 @@ static int l_Player_set_cruise_direction(lua_State *l)
 {
 	Player *player = LuaObject<Player>::CheckFromLua(1);
 	auto value = EnumStrings::GetValue("CruiseDirection", LuaPull<const char *>(l, 2));
-	assert(value != -1 && "Wrong set cruise direction");
+	if (value == -1)
+		luaL_error(l, "Wrong set cruise direction");
```

Comment:
As a minor nitpick because we do not (yet) automatically add context information to error messages, I'd appreciate if this message was formatted like so (pseudocode):
```suggestion
		luaL_error(l, "Player:SetCruiseDirection(): invalid cruise direction '%s' specified\n", mode);
```